cdap-ui/app/cdap/components/RulesEngineHome/RuleBookDetails/RulebookMenu/AddRulesEngineToPipelineModal.js

```diff
@@ -102,6 +110,7 @@ export default class AddRulesEngineToPipelineModal extends Component {
             artifact: realtimeHighestArtifact,
             config: {
               stages: [rulesEnginePlugin],
+              batchInterval: '10s',
```

Comment:
That was my assumption so removed it. I guess when we do `Object.assign` while importing a config we lose it. We need to consolidate what properties are defaulted in different pipeline types. Will work on it in a separate PR.